app/main.py

```diff
@@ -1 +1,36 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self, name: str, health: int = 100) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = False
+        Animal.alive.append(self)
+
+    def die(self) -> None:
+        if self in Animal.alive:
+            Animal.alive.remove(self)
+
+    def __repr__(self) -> str:
+        return (f"{{"
+                f"Name: {self.name}, "
+                f"Health: {self.health}, "
+                f"Hidden: {self.hidden}"
+                f"}}")
+
+    @classmethod
+    def __str__(cls) -> str:
+        return "[" + ", ".join(repr(animal) for animal in cls.alive) + "]"
```

Comment:
The `__str__` method is typically an instance method, not a class method. Consider renaming this method to something more appropriate for a class method, like `list_alive_animals`, or converting it to an instance method if it should represent an instance.